.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: java
+
+hosts:
+  - TRAVIS
+
+services:
+  - postgresql
+
+jdk:
+  - oraclejdk8
+
+install: mvn install -Dgpg.skip
+
+before_script:
+  - psql -c 'create database gemini_test;' -U postgres
+  - mvn archetype:update-local-catalog
+  - >
+      mvn archetype:generate -DarchetypeRepository=local -DarchetypeCatalog=local
+      -DarchetypeGroupId=com.techempower -DarchetypeArtifactId=gemini-resin-archetype
+      -Dpackage=foo.test -DartifactId=test-artifact-id -Dversion=1.0
+      -DmachineName=TRAVIS -DinteractiveMode=false
+      -DdatabaseHost=host.docker.internal
+      -DdatabaseName=gemini_test
+      -DdatabaseUsername=postgres
+  - cp .travis.conf test-artifact-id/src/main/webapp/WEB-INF/configuration/Base.conf
+  - cd test-artifact-id && mvn compile war:war
+  - curl -sL http://caucho.com/download/resin-4.0.56.tar.gz | tar xz --strip-components=1
+  - rm -rf webapps/*
+  - cp target/test-artifact-id-1.0.war webapps/ROOT.war
+
+script:
+  - java -jar lib/resin.jar console &
+  - sleep 10
+  - curl localhost:8080
```

Comment:
I think this needs to be changed a bit; maybe loop for a maximum of 30 seconds attempting to curl, eating the error (so travis doesn't error out), and returning 0 if curl is ever successful.

From the latest logs, it did not seem like 10s was enough for Gemini to become ready and accepting requests.